lib/releases.js

```diff
@@ -169,6 +174,31 @@ const generateChangeLog = (mergedPullRequests, config) => {
   return changeLog.join('').trim()
 }
 
+const resolveVersionKeyIncrement = (mergedPullRequests, config) => {
+  const priorityMap = {
+    patch: 1,
+    minor: 2,
+    major: 3
+  }
+  const labelToKeyMap = Object.fromEntries(
+    Object.keys(priorityMap)
+      .flatMap(key => [
+        config['version-resolver'][key].labels.map(label => [label, key])
+      ])
+      .flat()
+  )
+  const keys = mergedPullRequests
+    .filter(getFilterExcludedPullRequests(config['exclude-labels']))
```

Comment:
I am more inclined to merge #500 first, so you may need to rewrite this

Reply:
Flicking through that PR, think it'd be very easy to merge so happy for that approach (hopefully we don't get stuck though)